src/Exception/BadRequestParamHttpException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+/**
+ * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ */
+final class BadRequestParamHttpException extends BadRequestHttpException
```

Comment:
Not sure if another name or status code than 400 is better.

Reply:
As this exception is used when client errors are detected (like omitting or using a different type or value for a given param in the request), I think the name and the status code are good.